### PR TITLE
Park Tile

### PR DESCRIPTION
**Pull request by DCorp617**
Created enzyme test for ParkTile container

### CODE REVIEW

File: app/controllers/api/v1/parks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# stateID = params[:id]
+# parks = Park.where(state_id: stateID)
+#
+# wiki = ParseWiki.new
+# parks.each do |park|
+#   park.update_attributes(description: wiki.intro(park.name))
+# end
+#
+#
+# render json: Park.where(state_id: stateID)
```
Comment (AL6981) [inline app/controllers/api/v1/parks_controller.rb]: Please remove the commented out code.

File: app/models/parsewiki.rb [hunk 2]
```diff
@@ -2,15 +2,34 @@
...
+    self.class.base_uri "https://en.wikipedia.org/w/api.php?action=query&prop=pageimages&redirects=1&format=json&"
+    query_string = subject.gsub(' ','%20')
+    image_query = "titles=#{query_string}&pithumbsize=#{picture_size}"
+    wiki_object = self.class.get(image_query)
+    page_key = wiki_object['query']['pages'].keys[0]
+    wiki_image = wiki_object['query']['pages'][page_key]["thumbnail"]["source"]
+  end
+
+  def clean_text(wiki_intro)
+    #check to see if response has newline, indicating a paragraph. If so, extract only the first   paragraph
```
Comment (AL6981) [inline app/models/parsewiki.rb]: Remove comments.

File: app/models/parsewiki.rb [hunk 3]
```diff
@@ -2,15 +2,34 @@
...
     if (wiki_intro.include?("\n"))
       #return all indexes in which newline occurs
```
Comment (AL6981) [inline app/models/parsewiki.rb]: Remove comments

**Comment by adpaquin** [comment]
- Added Park show functionaility
 - Added tests to Park Show

File: app/javascript/react/container/ParkShowContainer.js [hunk 4]
```diff
@@ -1,20 +1,45 @@
...
-  constructor(props){
+  constructor(props) {
     super(props)
     this.state = {
       parkShow: {}
     }
   }
 
+  componentDidMount(){
+    let parkUrl = this.props.match.url
```
Comment (CassandraNewell) [inline app/javascript/react/container/ParkShowContainer.js]: clever!

File: spec/features/controllers/park_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    describe "GET #show" do
+
+    let!(:state) { FactoryBot.create(:state) }
+    let!(:park) { FactoryBot.create(:park) }
+
+    context "when user goes to park show page" do
+      before {get :show, params: { id: 42 , state_id: 1} }
+      
+      it "returns a specific park" do
+        get :show, params: { id: 42 , state_id: 1}
```
Comment (CassandraNewell) [inline spec/features/controllers/park_controller_spec.rb]: Because you have this in a before, you shouldn't need lines 18 or 23

Comment (adpaquin) [reply spec/features/controllers/park_controller_spec.rb]: Deleted these lines

**Pull request merged**